mlflow/langchain/__init__.py

```diff
@@ -967,10 +967,6 @@ def autolog(
     log_model_signatures=False,
     log_models=False,
     log_datasets=False,
-    # TODO: log_inputs_outputs was originally defaulted to True in the production version of
-    # the LangChain autologging, as it is a common use case to log input/output table for
-    # evaluation. Once tracing is fully launched, this should be supported by the tracer
-    # but we should design it to be compatible with the existing UJ.
```

Comment:
We don't need to fix this anymore in my opinion, we should probably drop this in the future version. Logging inputs & outputs is error-prone when converting to dataframe.